torch/csrc/api/src/nn/modules/loss.cpp

```diff
@@ -77,5 +77,18 @@ Tensor CosineEmbeddingLossImpl::forward(
   return F::cosine_embedding_loss(input1, input2, target, options);
 }
 
+/ ============================================================================
```

Comment:
We are missing a `/` here:
```suggestion
// ============================================================================
```